packages/scene-composer/src/layouts/SceneLayout/components/FoldableContainer.scss

```diff
@@ -0,0 +1,26 @@
+// @use '~/@awsui-design-tokens/polaris.scss';
+
+.tm-wrapper-left,
+.tm-wrapper-right {
+  display: flex;
+  align-items: stretch;
+  height: 100%;
+}
+
+.tm-wrapper-left { flex-direction: row;}
```

Comment:
Looks like you formatter might be off? Referring to the `;}` instead of `; }`.

Reply:
Great catch! Maybe this is why the linter is failing. :/

Reply:
Ah... hadn't read the overall comment yet.